rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -28,11 +28,50 @@ const Landing = () => {
   });
   const [currentRigSelection, setCurrentRigSelection] = useState('');
   const [newlyCreatedRig, setNewlyCreated] = useState('');
-  const [wellModalOpen, setWellModalOpen] = useState(false)
+  const [wellModalOpen, setWellModalOpen] = useState({
```

Comment:
wellModalOpen sounds like a boolean, this has been changed to an object so lets update the name to reflect that.